templates/taxi/driver_list.html

```diff
@@ -6,7 +6,9 @@
     <ul>
       {% for driver in driver_list%}
         <li>
-          <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
+          <a href="{% url "taxi:driver-detail" pk=driver.id %}">
+              {{ driver.username }} {% if driver == current_user %}(Me){% endif %}
```

Comment:
Ensure that `driver.username` is correctly linked to the `Driver` model. If `username` is not a field on the `Driver` model, consider using the appropriate field.